drivers/misc/siklu_dma_cache.c

```diff
@@ -78,14 +78,14 @@ static long etx_ioctl(struct file *file, unsigned int cmd, unsigned long arg) {
 				return -EFAULT;
 			}
 			if ((enum dma_data_direction)args.dir == DMA_TO_DEVICE) {
-				arch_sync_dma_for_device(NULL, args.paddr,
-						args.len, DMA_TO_DEVICE);
+				arch_sync_dma_for_device(args.paddr, args.len,
+						DMA_TO_DEVICE);
```

Comment:
This fixes should be squashed into the previous commit.

Reply:
Ok

Reply:
Squashed. Please re-review.